sanskrit_parser/lexical_analyzer/sanskrit_lexical_analyzer.py

```diff
@@ -53,20 +58,21 @@ def hasNode(self, t):
                boolean
         '''
         return t in self.G
-    def appendToNode(self, t, rdag):
-        """ append rdag to self, adding edges from a given node to rdag's roots 
+
+    def appendToNode(self, t, nodes):
+        """ Create edges from t to nodes
 
             Params:
                 t (SanskritObject)      : Node to append to
-             rdag (SanskritLexicalGraph): Graph to append to node
+                nodes (iterator(nodes)) : Nodes to append to t
         """
         # t is in our graph
         assert t in self.G
-        self.G = nx.compose(self.G, rdag.G)
-        for r in rdag.roots:
+        for r in nodes:
             self.G.add_edge(t, r)
```

Comment:
I think this idea will work!
Need to update some comments
`# Make sure we got a graph back` 
Should be  "set"

Reply:
The travis tests are passing (except for the one spurious failure on the py2 in the PR test, but the original check-in passes and py3 run passed for the PR ). I have also done some tests locally, so I am pretty confident that these changes work.